github/GithubIntegration.py

```diff
@@ -252,7 +252,11 @@ def create_jwt(self, expiration: int | None = None) -> str:
         return self.auth.create_jwt(expiration)
 
     def get_access_token(
-        self, installation_id: int, permissions: dict[str, str] | None = None
+        self,
+        installation_id: int,
+        permissions: dict[str, str] | None = None,
+        repositories: list[str] | None = None,
+        repository_ids: list[int] | None = None,
```

Comment:
We could merge these to into
```suggestion
        repositories: list[str | int] | None = None,
```
to simplify calling this method for users.